frontend/demo/src/components/ColorSelector.module.scss

```diff
@@ -0,0 +1,166 @@
+@import '../assets/scss/colors.scss';
+
+.colorSelector {
+  position: relative;
+  overflow: hidden;
+  div {
+    display: inline-block;
+    width: 24px;
+    animation-timing-function: cubic-bezier(0, 1, 1, 0);
+    margin: 0 16px 0 0;
+  }
+
+  input[type='radio'] {
```

Comment:
Nitpick: It would be great if the selectors in this file were broken down into classes. Right now it's hard for a new reader of the code to understand what they mean.